packages/client/hmi-client/src/workflow/ops/stratify-mira/tera-stratify-mira.vue

```diff
@@ -105,6 +124,9 @@ const modelNodeOptions = ref<string[]>([]);
 const teraModelDiagramRef = ref();
 const newModelName = ref('');
 
+const editor = ref<VAceEditorInstance['_editor'] | null>(null);
```

Comment:
Does `editor` need to be a ref? I am always a little cautious about sticking 3rd party stuff down into refs cause you never quite know how they would behave when they become reactive.

Reply:
done